### PR TITLE
Handle Stop Date Changed

### PR DESCRIPTION
**Pull request by samuel-dev-dba**

### CODE REVIEW

File: src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Steps/ApprovalsInterfaceSteps.cs [hunk 1]
```diff
@@ -392,7 +394,23 @@ public void WhenTheApprovalsServiceNotifiesThePaymentsServiceOfThatTheApprentice
...
             }
         }
-
+        
+        [When(@"the Approvals service notifies the Payments service that the apprenticeships stop date has changed")]
+        public void WhenTheApprovalsServiceNotifiesThePaymentsServiceThatTheApprenticeshipsStopDateHasChanged()
+        {
+            foreach (var approvalsApprenticeship in ApprovalsApprenticeships)
+            {
+                var (employer, sendingEmployer, provider, learner) = GetApprovalsReferenceData(approvalsApprenticeship);
```
Comment (davehowlett) [inline src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Steps/ApprovalsInterfaceSteps.cs]: the results of this method aren't being used. Should they be?

File: src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Steps/ApprovalsInterfaceSteps.cs [hunk 2]
```diff
@@ -415,6 +433,9 @@ private ApprenticeshipModel CreateApprenticeshipModel(ApprovalsApprenticeship ap
                     ? DateTime.UtcNow
                     : apprenticeshipSpec.AgreedOnDate.ToDate(),
                 IsLevyPayer = true,
+                StopDate = string.IsNullOrWhiteSpace(apprenticeshipSpec.StoppedOnDate)
```
Comment (davehowlett) [inline src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Steps/ApprovalsInterfaceSteps.cs]: You've just introduced a new extension method that does exactly this. I understand why you don't want to change existing steps, but this is new.

File: src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Steps/ApprovalsInterfaceSteps.cs (hunk 1 above)
Comment (samuel-dev-dba) [reply src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Steps/ApprovalsInterfaceSteps.cs]: removed

File: src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Steps/ApprovalsInterfaceSteps.cs (hunk 2 above)
Comment (samuel-dev-dba) [reply src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Steps/ApprovalsInterfaceSteps.cs]: changed

**Comment by samuel-dev-dba** [comment]
Closing in favour of PR from a branch -- PV2-1030_PV2-1031_PV2-695_PV2-694_Approvals_Handle_Pause_And_Resume

**Pull request closed without merging**